build-tools/cache_samples.sh

```diff
@@ -99,11 +99,27 @@ function cache_devfile() {
     fi
 }
 
-devfileEntriesFile=$1
-samplesDir=$2
+if [[ "$OSTYPE" == "darwin" ]]
```

Comment:
```suggestion
if [[ "$OSTYPE" == "darwin"* ]]
```

I noticed it wasn't hitting the condition properly and I found if I run `echo $OSTYPE` I was receiving `darwin23.0`, adding the `*` allows for it to parse darwin properly if version is attached to it. Another option is to use `"$(uname -s)" == "Darwin"`

Reply:
resolved with https://github.com/devfile/registry-support/pull/239/commits/c1320acb37e95268aa78d0f8c4cd74669b1b52f6